cmd/minikube/cmd/start.go

```diff
@@ -294,6 +294,16 @@ func provisionWithDriver(cmd *cobra.Command, ds registry.DriverState, existing *
 		validateDockerStorageDriver(driverName)
 	}
 
+	k8sVersion := getKubernetesVersion(existing)
+
+	// Disallow accepting addons flag without Kubernetes
+	// It places here because cluster config is required to get the old version.
+	if cmd.Flags().Changed(config.AddonListFlag) {
+		if k8sVersion == constants.NoKubernetesVersion || viper.GetBool(noKubernetes) {
+			exit.Message(reason.Usage, "Cannot enable addons without Kubernetes")
```

Comment:
how about add a message here that tells user what to do.
"You can not enable this addon with a cluster without Kubernetes, to re-configure your cluster with Kubernetes run this command
minikube start --no-kubernetes=false
 "

Reply:
Much better. I appreciate it. I just re-request a review.

Reply:
The problem is `--no-kubernetes=false` won't actually re-enable Kubernetes. I'd recommend we change the command to `--kubernetes-version=stable`